todo_app/data/trello_items.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import os
+
+Board_ID = os.environ.get('BOARD_ID')
+
+def build_url(endpoint):
+    return 'https://api.trello.com/1/'+endpoint
+
+def get_auth_params():
+    return({'key':os.environ.get('KEY'), 'token':os.environ.get('TOKEN')})
+
+def build_params(params = {}):
+    full_params = get_auth_params()
+    full_params.update(params)
+    return full_params
+
+def fetch_all_cards():
+    r_json = json_of_all_cards()
+    return r_json
+
+def create_new_card(title):
+    url = build_url('cards')
+    data_todo_list = (define_lists_in_board())[0]
+    query = build_params({'name':title,'idList':data_todo_list['idList']})
+    headers = {
+        "Accept": "application/json"
+    }
+    requests.request("POST", url, params=query, headers=headers)
+
+def mark_as_complete(card_id):
+    url = build_url('cards/'+card_id)
+    data_done_list = (define_lists_in_board())[-1]
+    query = build_params({'idList':data_done_list['idList']})
+    headers = {
+        "Accept": "application/json"
+    }
+    requests.put(url, params=query, headers=headers)
+
+def json_of_all_cards():
+    url = build_url('boards/'+Board_ID + '/cards/open')
```

Comment:
I would advise using an f string here: 
```suggestion
    url = build_url(f"boards/{Board_ID}/cards/open")
```